index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML 5 My Portfolio</title>
+    <link rel="stylesheet" href="style.css"/>
+  </head>
+  <body class="bg">
+  <div class="wrapper">
+    <div class="toolbar">
+      <div class="mylogo">My Logo</div>
+      <div class='hamburger'>
+        <div class='line'></div>
+        <div class='line'></div>
+        <div class='line'></div>
+      </div>
+    </div>
+     	<p class="myheader">I’m PRATAP Glad to see you!</p>
+    	<p class="content">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    	<p class="connect">LET’S CONNECT</p>
```

Comment:
- [ ] This element should be  a link `<a>` or `<button>` tag instead of `<p>` tag.
- [ ] And according to the requirements, You should Implement the button interactions (enable, hover, etc.). so I suggest adding a hover on the `connect` class.

For example
```
.connect:hover {
  color: #fff;
  background-color: #7f8cff;
}
```

![image](https://user-images.githubusercontent.com/45231276/176402228-2c9255aa-c65f-4928-bbaf-375ec41b19ff.png)